x-pack/legacy/plugins/apm/public/services/rest/callApi.ts

```diff
@@ -55,6 +58,34 @@ export async function callApi<T = void>(
   return res;
 }
 
+export const callApmApi: APICall<typeof api> = ({
+  method,
+  pathname,
+  params
+}: {
+  method: any;
+  pathname: string;
+  params?: {
+    [key in keyof RouteParams]: any;
+  };
+}) => {
+  const path = idx(params, _ => _.path) || {};
+  const query = idx(params, _ => _.query) || {};
+  const body = idx(params, _ => _.body) || {};
+
+  const formattedPathname = pathname
+    .split('/')
+    .map(part => (part.startsWith(':') ? path[part.substr(1)] : part))
```

Comment:
Didn't you end up using `{variable}` instead of `:variable`?

Reply:
Yeah, my bad. I've dropped path parameters now so this is no longer relevant.